### DISCUSSION TITLE
Prove

### DISCUSSION THREAD

**Discussion by Yigitfdn** [community/community#70218] [GitHub Education]
### Hi everyone,

How can ı prove myself to github for my current student status? Neither my student card or any other document doesnt have any date because my student validity still valid until I graduate.

**Comment by FilipJaskovic** [comment]
Try this:


1.	Upload a Current Transcript:
Submitting a recent academic transcript that shows your active enrollment in courses can serve as proof of your current student status .
2.	Letter on School Letterhead:
A letter from your school on official letterhead confirming your current enrollment can also be used to prove your student status. This letter can be from a faculty member or an administrative official from your school .
3.	Enrollment Verification Letter:
Similar to a letter on school letterhead, an enrollment verification letter from your school can also be used as proof of your current student status .
4.	Class Schedule:
A class schedule showing your current or upcoming classes can also serve as proof of your current student status .
5.	School-Issued Email Verification:
If your school has issued you an email address, you can use this email address to verify your student status on GitHub. Note that the email address needs to be verifiable and linked to your educational institution .
6.	Contact GitHub Education Support:
If all other methods fail, you can submit a ticket to GitHub Education Support explaining your situation and seeking assistance on how to prove your student status .
